docs/rules/no-plusplus.md

```diff
@@ -1,4 +1,4 @@
-# Disallow ++ and -- (no-plusplus)
+# disallow the unary operators `++` and `--` (no-plusplus)
 
 The `no-plusplus` rule flags the use of unary operators, `++` and `--`.
```

Comment:
Suggest delete the first sentence and neutral code example, and then simplify the paragraph preceding the 2 contrasting code examples to something like:

Because the unary `++` and `--` operators are subject to automatic semicolon insertion, differences in whitespace can change semantics of source code.